docs/contributing/deciding-pr-high-impact.md

```diff
@@ -23,6 +22,8 @@ On this page, we will share some guidelines to help you assess the impact degree
 - It's a **big PR** (i.e. adds several changes in many files).
 - It has **i18n changes** (for example, any file from `woocommerce/i18n` is modified).
 
+Note that security issues and fixes should all go through our [security reporting process](./reporting-security-issues.md).
```

Comment:
With all the bolding going on above, maybe let's do something similar here as well? 

```suggestion
Note that **security issues and fixes** should all go through our [security reporting process](./reporting-security-issues.md).
```